src/rules/no-unused-modules.js

```diff
@@ -228,7 +236,7 @@ const prepareImportsAndExports = (srcFiles, context) => {
         }
         const localImport = imports.get(key) || new Set();
         value.declarations.forEach(({ importedSpecifiers }) =>
-          importedSpecifiers.forEach(specifier => localImport.add(specifier))
+          importedSpecifiers.forEach(specifier => localImport.add(specifier)),
```

Comment:
I think it was added by the following configuration.
https://github.com/import-js/eslint-plugin-import/blob/master/src/.eslintrc.yml

Reply:
yes, but not on function arguments, which is the case here.

Reply:
I didn't understand what you meant. (Maybe it's because I'm not familiar with English...)
Are you saying I shouldn't fix it even if ESLint warns me?

https://github.com/import-js/eslint-plugin-import/runs/3345983029?check_suite_focus=true

![image](https://user-images.githubusercontent.com/16508807/129671959-3afcf675-3f52-49d4-86cd-ca2bc1743164.png)

Reply:
yes, those are just warnings, not errors.

Reply:
Thank you! I understand. Revert comma changes.

Reply:
I made change to this PR https://github.com/import-js/eslint-plugin-import/pull/2191/commits/b45d9ecb15d0dc28066176585c4fe88665e007d7